src/page.js

```diff
@@ -202,10 +202,15 @@ export default function wikiPage(rawPageInfo, apiOptions) {
 	 * @example
 	 * new Wiki().page('Batman').then(page => page.info('alter_ego'));
 	 * @method WikiPage#info
-	 * @param  {String} [key] - Information key
+	 * @param  {String} [key] - Information key. Falsy keys are ignored
+   * @param  {Object} [options] - Infobox parser options
 	 * @return {Promise} - info Object contains key/value pairs of infobox data, or specific value if key given
 	 */
-	function info(key) {
+	function info(key, options) {
+    if (key && typeof key === 'object') {
```

Comment:
Let's go for one place for settings. If you like though, you can added another property in the settings for the parser... ex.

```js
const settings = {
  apiUrl: '...',
  parser: {
    simplifyDataValues: false,
  }
}
```